src/Components/leftbar/components/ReactRangeSlider.tsx

```diff
@@ -20,61 +23,64 @@ const ReactRangeSlider = ({
   };
    
   return (
-    <div className="mt-10 px-2 w-full">
-      <div>
-        <h1>Արժեք</h1>
-      </div>
-      <div className="flex justify-between items-center my-4 flex-wrap">
-        <div className="flex items-center my-2">
-          <p className="font-[600] mr-[5px] text-[14px]">Մին</p>
-          <h2 className="border py-[5px] md:px-[10px] lg:px-[20px] ">{pricesRange[0].currentPrice}</h2>
-        </div>
-        <div className="flex items-center">
-          <p className="font-[600] mr-[5px] text-[14px]">Մաքս</p>
-          <h2 className="border py-[5px] md:px-[10px] lg:px-[20px]">{pricesRange[1].currentPrice}</h2>
-        </div>
-      </div>
-        <Range
-          step={100}
-          min={pricesRange[0].borderPrice || 0}
-          max={pricesRange[1].borderPrice || 1}
-          values={range}
-          onChange={(values) => setPricesRange(state=>{
-            return state.map((elem:any,index)=>({
-               ...elem,currentPrice:values[index]
-            }))
-          })}
-          onFinalChange={onFinalChange}
-          renderTrack={({ props, children }) => (
-            <div
-              {...props}
-              style={{
-                ...props.style,
-                height: "2px",
-                width: "100%",
-                backgroundColor: "#6e6e",
-                outline: "none",
-              }}
-            >
-              {children}
-            </div>
-          )}
-          renderThumb={({ props }) => (
-            <div
-              {...props}
-              style={{
-                ...props.style,
-                height: "15px",
-                width: "15px",
-                borderRadius: "50%",
-                backgroundColor: "cyan",
-                outline: "none",
-                cursor: "pointer",
-              }}
-            />
-          )}
-        />
-    </div>
+    <>
+      {homePageTexts?.value.description && (
+         <div className="mt-10 px-2 w-full">
+         <div>
+           <h1>{homePageTexts?.value.description}</h1>
+         </div>
+         <div className="flex justify-between items-center my-4 flex-wrap">
+           <div className="flex items-center my-2">
+             <h2 className="border py-[5px] md:px-[10px] lg:px-[20px] ">{pricesRange[0].currentPrice}</h2>
+           </div>
+           <div className="flex items-center">
+             <h2 className="border py-[5px] md:px-[10px] lg:px-[20px]">{pricesRange[1].currentPrice}</h2>
+           </div>
+         </div>
+           <Range
+             step={100}
+             min={pricesRange[0].borderPrice || 0}
+             max={pricesRange[1].borderPrice || 1}
+             values={range}
+             onChange={(values) => setPricesRange(state=>{
```

Comment:
```suggestion
             onChange={(values) => setPricesRange(state=>{
```

move this callback to  a separate function